### ISSUE TITLE
Add more themes

### ISSUE THREAD

**Issue by unoffending** [opentoonz/opentoonz#5395]
### Is your feature request related to a problem? Please describe

When you use one theme for a long time, you can get tired of it... Also, many people like other colors, such as purple...
P.S. Please don't ban me

### Describe the solution you'd like

I suggest making more themes in OpenToonz, different colors like blue, orange, pink...
I know it sounds strange, but it's just an idea...

**Comment by RodneyBaker** [comment]
This is something every user can do.
In fact, theme from Tahoma2D can be used with Opentoonz and vice versa as well.

You should join one of the Opentoonz discords where they share themes.
Those themes can be added to the normal theme location in the stuff directory or copy/pasted into the Additional Stylesheets area of Preferences for use.

**Comment by unoffending** [comment]
> This is something every user can do. In fact, theme from Tahoma2D can be used with Opentoonz and vice versa as well.
> 
> You should join one of the Opentoonz discords where they share themes. Those themes can be added to the normal theme location in the stuff directory or copy/pasted into the Additional Stylesheets area of Preferences for use.

Thank you!

**Comment by unoffending** [comment]
> This is something every user can do. In fact, theme from Tahoma2D can be used with Opentoonz and vice versa as well.
> 
> You should join one of the Opentoonz discords where they share themes. Those themes can be added to the normal theme location in the stuff directory or copy/pasted into the Additional Stylesheets area of Preferences for use.

I haven't found any OpenToonz groups on discord... Looks like they can only be joined by invitation

**Comment by RodneyBaker** [comment]
Here's a invite link to the main Opentoonz discord:

Link:  https://discord.gg/824Y6rqTWR

There are other discords and venues but most of the people in those also hang out in the one at this link as well.

**Comment by unoffending** [comment]
> Here's a invite link to the main Opentoonz discord:
> 
> Link: https://discord.gg/824Y6rqTWR
> 
> There are other Opentoonz related discords and venues but most of the people in those also hang out in the one at this link as well.

Thank you!!

**Comment by RodneyBaker** [comment]
I've added what themes I could find to the ExtraStuff repo on github.
There we are starting to collect some resources for Opentoonz, Tahoma2D and the like.

Link:  https://github.com/OpenAnimationLibrary/extrastuff/tree/master/ot/stuff/config/themes

As for adding themes to the Opentoonz install... that's mostly just a matter of someone submitting those themes for inclusion.
I suppose at some point we'd have to limit the number of themes but at this time that isn't a problem.

Will plan to close this request as a general request demonstrating interest in having more themes.
Those apt to create those additonal themes and customizations should take note.
In general this is a matter for annual review... the annual UI scrub... where we either decide what is there now is sufficient or discuss how best to move forward.

In other words, now is a good time to discuss such things.

**Comment by unoffending** [comment]
> This is something every user can do. In fact, theme from Tahoma2D can be used with Opentoonz and vice versa as well.
> 
> You should join one of the Opentoonz discords where they share themes. Those themes can be added to the normal theme location in the stuff directory or copy/pasted into the Additional Stylesheets area of Preferences for use.

I think that creating a theme yourself is quite difficult... Still, the qss file is large...
Therefore, I still think that a larger number of themes would not harm OpenToonz...

**Comment by unoffending** [comment]
> I've added what themes I could find to the ExtraStuff repo on github. There we are starting to collect some resources for Opentoonz, Tahoma2D and the like.
> 
> Link: https://github.com/OpenAnimationLibrary/extrastuff/tree/master/ot/stuff/config/themes
> 
> As for adding themes to the Opentoonz install... that's mostly just a matter of someone submitting those themes for inclusion. I suppose at some point we'd have to limit the number of themes but at this time that isn't a problem.
> 
> Will plan to close this request as a general request demonstrating interest in having more themes. Those apt to create those additonal themes and customizations should take note. In general this is a matter for annual review... the annual UI scrub... where we either decide what is there now is sufficient or discuss how best to move forward.
> 
> In other words, now is a good time to discuss such things.

Thank you!

**Comment by RodneyBaker** [comment]
Will close this as adding themes is one of those recurring updates that just requires someone to submit a new theme.
